modules/serverless-rn-1-12-0.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-12-0_{context}"]
+
+= Release Notes for Red Hat {ServerlessProductName} 1.12.0
+
+[id="new-features-1-12-0_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 0.18.2.
+* {ServerlessProductName} uses Knative Eventing 0.18.6.
+* {ServerlessProductName} now uses Kourier 0.18.0.
+* {ServerlessProductName} now uses Knative `kn` CLI 0.18.4.
+* {ServerlessProductName} now uses Knative Kafka 0.18.0.
+
+[id="fixed-issues-1-12-0_{context}"]
+== Fixed issues
```

Comment:
Shouldn't we list all CVE's we are fixing with this release?

Reply:
We don't generally, it's listed in the errata already?